lightning/src/chain/channelmonitor.rs

```diff
@@ -2531,9 +2581,11 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitorImpl<Signer> {
 		ret
 	}
 
-	pub fn get_and_clear_pending_events(&mut self) -> Vec<Event> {
+	/// Gets the set of events which are repeated regularly (i.e. those which RBF bump
```

Comment:
Use "e.g." if this is just one potential example of such an event (assuming there may be others in the future).